test/array-as-properties/index.ts

```diff
@@ -1,14 +1,15 @@
 interface SquareConfig {
-  color: string
-  width: number
+  color: string;
+  width: number;
 }
 
 interface SquareConfigsInterface {
-  squares: SquareConfig[]
+  squares: SquareConfig[];
+  logs?: string[];
```

Comment:
After adding this you also have to update `test/array-as-properties/fixture.json` accordingly.
Did you try to run the tests?